memif-attach/main.go

```diff
@@ -186,6 +186,23 @@ func dumpMemif(ctx context.Context, conn api.Connection) () {
 	return 
 }
 
+func handleHelloMsg(ctx context.Context, conn_client net.Conn) {
+	helloMsg := &HelloMsg{}
+	reader := bufio.NewReader(conn_client)
+	err := binary.Read(reader, binary.BigEndian, helloMsg)
+	if err != nil {
+		log.Entry(ctx).Fatalln("ERROR: read from VPP master hello message failed:", err)
+	}
+	log.Entry(ctx).Infof("max_region: %v\n"+
```

Comment:
INFO[0000] max_region: 512
Max_m2s_ring: 22096
Max_s2m_ring: 20512
Min_version: 12848
Max_version: 11824
Max_log2_ring_size: 57
Name: [45 114 101 108 101 97 115 101 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 2 0 2 0 1 1 0 0]

Wonder if return value is reasonable or not?